### ISSUE TITLE
Version Issue

### ISSUE THREAD

**Issue by makiisthenes** [vv1ck/API-snapchat#2]
**I got this issue when running.**

`{"logged":false,"message":"أنت تستخدم إصدارًا من Snapchat لم يعد مدعومًا. يُرجى التحديث إلى أحدث إصدار للتطبيق لاستخدام Snapchat. شكرًا."}
`
_You're using a version of Snapchat that's no longer supported. Please update to the latest version of the app to use Snapchat. Thank you._

----

**I changed the user-agent of the header to the latest version:**

` 'User-Agent': 'Snapchat/11.41.0.36 (iPhone9,3; iOS 14.4.1; gzip)',`
----
**But got another error saying:**

`{"logged":false,"message":"عفوًا! أنت تحاول الوصول إلى Snapchat بواسطة تطبيق خاص بجهة خارجية! يُرجى استخدام تطبيق Snapchat الرسمي بدلاً من ذلك لو سمحت . 👻"}`

_You're welcome! You're trying to access Snapchat with a third-party app! Please use the official Snapchat app instead, please. 👻_

**Comment by vv1ck** [comment]
Yes, unfortunately, this program has stopped working and needs to be
updated. I do not have enough time to update it

You can use this program, (
https://github.com/vv1ck/Snapchat-Login
) it works fine, and the captcha is also skipped



*I got this issue when running.*

**Comment by Tt7829** [comment]
****
